docs/linking_to_external_code/import_maps.md

```diff
@@ -60,7 +60,7 @@ import { MyUtil } from "/util.ts";
 
 You may map a different directory: (eg. src)
 
-```json
+```json5
```

Comment:
There is no good reason that there is a comment in this block, and should be moved outside instead.

Reply:
I see. I moved and made them bold like previous code blocks.